scripts/data/credentials.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+#
+# Usage:
+# ./scripts/data/credentials.sh
+#
+# To create the records on a Primaza server which is not localhost:8080
```

Comment:
I would say: to create the records on a different Primaza server

Reply:
Fixed with a70f38c